.github/workflows/new-pull-request.yml

```diff
@@ -5,6 +5,30 @@ on:
     types: [opened]
 
 jobs:
+  name: Greetings
+  on: [pull_request_target]
+
+  jobs:
+    greeting:
+      runs-on: ubuntu-latest
+      permissions:
+        issues: write
+        pull-requests: write
+      steps:
+      - uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          pr-message: >
+            Thank you for submitting your first pull request (PR) to PlasmaPy!  We invite you to check out our [**Code of Conduct**](https://docs.plasmapy.org/en/stable/CODE_OF_CONDUCT.html) and [**contributor guide**](https://docs.plasmapy.org/en/latest/contributing/index.html). 🥳
+
+            Please **add yourself to the list of contributors** in [docs/about/credits.rst](https://github.com/PlasmaPy/PlasmaPy/blob/main/docs/about/credits.rst) to make sure that you get credit for your contribution. 🔖
```

Comment:
For the sake of new contributors, maybe specify that this means to modify that file within you PR

Reply:
Very good clarification to include!